paddle/framework/selected_rows.cc

```diff
@@ -12,5 +12,45 @@ limitations under the License. */
 #include "paddle/framework/selected_rows.h"
 
 namespace paddle {
-namespace framework {}  // namespace framework
+namespace framework {
+void SelectedRows::SerializeToStream(std::ostream &os,
+                                     const platform::DeviceContext &dev_ctx) {
+  PADDLE_ENFORCE_NOT_NULL(
+      value_, "serialize SelectedRows failed since Tensor is nullptr.");
+  value_->SerializeToStream(os, dev_ctx);
+  {
```

Comment:
version field

Reply:
It has also been serialized in the `Tensor::SerializeToStream`  .